docs/creating-new/create/cmake.rst

```diff
@@ -452,19 +449,25 @@ On Windows:
 
 .. code-block:: none
 
-  > cd hunter
+  > where python
+  C:\Python3\python.exe
 
-  [hunter]> where polly.py
-  C:\...\bin\polly.py
+  > git clone https://github.com/ruslo/polly
+  > cd polly
+  [polly]> set PATH=%CD%\bin;%PATH%
 
-  [hunter]> polly.py --help
+  > cd hunter
+  [hunter]> where polly.bat
+  C:\...\bin\polly.bat
+
+  [hunter]> polly.bat --help
   Python version: 3.5
   usage: polly.py [-h]
   ...
 
   [hunter]> set TOOLCHAIN=vs-12-2013
   [hunter]> set PROJECT_DIR=examples\hunter_box_1
-  [hunter]> .\jenkins.py
+  [hunter]> python jenkins.py
```

Comment:
Using `.\jenkins.py` works fine. You should tune your file associations, see the Stackoverflow question below.